### ISSUE TITLE
Improve handling of the exception that are generated when a blob has been deleted

### ISSUE THREAD

**Issue by SandGrainOne** [Altinn/altinn-file-scan#100]
### Description

Blobs being deleted before we start a file scan occasionally results in Internal Server Error in the file scan application.

In this specific case the file scan application should return Ok instead. The situation should still be logged, but the log level should be Warning instead of Error.

### Additional Information

This should be improved to avoid unnecessary alerts. There are currently an alert rule if the file scan application responds to requests with Internal Server Error.

**Comment by acn-sbuad** [comment]
duplicate of #43. Closing
